tests/e2e/framework/kubernetes.go

```diff
@@ -36,7 +36,6 @@ const (
 	nonAuthInstallFile = "istio-one-namespace.yaml"
 	authInstallFile    = "istio-one-namespace-auth.yaml"
 	istioSystem        = "istio-system"
-	mixerConfigDefault = "istio-config-default"
```

Comment:
can we keep this around and set same value as istio-system? seems like more future-proof?

Reply:
It's not hard to put back, if needed. Let's try to keep the code clean, otherwise we might not get back to it.